### DISCUSSION TITLE
Support for mouse input, not only touch

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/maui#246] [General]
Windows, Mac, and Linux use mouse input most of the time. We need support for mouse input in MAUI, not only touch. Raising mouse click events as touch events is not good enough.

**Comment by charlesroddie** [comment]
Can you spec this out @aaabbbcccddd00001111 ?

**Comment by lytico** [comment]
something like this:
https://github.com/xamarin/xamarin-forms-samples/tree/main/Effects/TouchTrackingEffect

+ enhanced with:
https://github.com/xamarin/xamarin-forms-samples/blob/main/Effects/TouchTrackingEffect/TouchTrackingEffect/TouchTrackingEffect/TouchActionEventArgs.cs

```
 public class TouchActionEventArgs { 
...
       TouchDevice Device { get; }
}

public enum TouchDevice {
        Mouse,
        Finger,
        ...
}
```
